### PR TITLE
Add description for e-book format support

### PR DESCRIPTION
**Pull request by clockr63**
### Change Description ###

 - Add description for e-book format support 

### API Changes ###

 - ACR: ACR-1635, ACR-1643

### CODE REVIEW

File: docs/application/native/guides/multimedia/media-content.md [hunk 1]
```diff
@@ -1763,6 +1763,18 @@ For metadata of an audio file, call the `media_info_get_audio()` function with t
 | `Channel`        | Channel information for the audio content |
 | `Duration`       | Duration of the audio content            |
 
+For metadata of an book file, call the `media_info_get_book()` function with the media handle.
```
Comment (annie-samsung) [inline docs/application/native/guides/multimedia/media-content.md:1766]:
```suggestion
For metadata of a book file, call the `media_info_get_book()` function with the media handle.
```

**Comment by annie-samsung** [comment]
@clockr63, this entire document seems to follow a flat structure. Since it is a very big document, I would suggest following a nested structure.

image

File: docs/application/native/guides/multimedia/media-content.md (hunk 1 above)
Comment (clockr63) [reply docs/application/native/guides/multimedia/media-content.md]: Done.

**Comment by annie-samsung** [comment]
@clockr63, you thoughts on this comment: https://github.com/Samsung/tizen-docs/pull/1558#issuecomment-976605779

**Comment by clockr63** [comment]
> @clockr63, you thoughts on this comment: [#1558 (comment)](https://github.com/Samsung/tizen-docs/pull/1558#issuecomment-976605779)

Please check below commit:
https://github.com/Samsung/tizen-docs/pull/1558/commits/d6a885b2c84dc8596b83af8e064dd96956fac598

**Review by annie-samsung — APPROVED** [review]
looks good

**Pull request merged**